### PR TITLE
Commerce 8885

### PR DESCRIPTION
**Pull request by garofalog**
https://issues.liferay.com/browse/COMMERCE-8885

### CODE REVIEW

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/info_box/js/info_box.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ */
+
+export default function ({
+	actionLabelValidator,
+	actionTargetId,
+	actionTargetIdValidator,
+	linkId,
+}) {
+	const link = document.getElementById(linkId);
+	const actionIsValid =
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/info_box/js/info_box.js:22]: Rename it to "valid"

File: modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/info_box/start.jsp [hunk 2]
```diff
@@ -50,4 +35,19 @@ String linkId = PortalUtil.generateRandomKey(request, "info-box") + "_action-lin
...
 	</header>
 
-	<div class="description">
\ No newline at end of file
+	<div class="description">
+
+<liferay-frontend:component
+	context='<%=
+		HashMapBuilder.<String, Object>put(
+			"actionLabelValidator", actionLabel
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/info_box/start.jsp:43]: You can remove it and validate the actionLabel directly into the component.

File: modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/info_box/start.jsp [hunk 3]
```diff
@@ -50,4 +35,19 @@ String linkId = PortalUtil.generateRandomKey(request, "info-box") + "_action-lin
...
+		).put(
+			"actionTargetId", HtmlUtil.escapeJS(actionTargetId)
+		).put(
+			"actionTargetIdValidator", actionTargetId
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/info_box/start.jsp:47]: Same here, you can validate it in the component.

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@garofalog Please take a look at the comments, thanks!

**Review by alessiorendina — APPROVED** [review]

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
@garofalog why did you launch a new relevant build? We just need the QA check here.

**Comment by alessiorendina** [comment]
ci:test:relevant

**Pull request closed without merging**